04 Strategy Library/162 Momentum Effect in Stocks in Small Portfolios/02 Method.html

```diff
@@ -0,0 +1,68 @@
+<p>
+  The investment universe consists of all US listed companies. Stocks which have no fundamental data are ruled out from the universe.
+</P>
+<div class="section-example-container">
+<pre class="python">
+def CoarseSelectionFunction(self, coarse):
+    if self.yearly_rebalance:
+        # drop stocks which have no fundamental data
+        self.filtered_coarse = [x.Symbol for x in coarse if (x.HasFundamentalData)]
+        return self.filtered_coarse
+    else:
+        return []
+</pre>
+</div>
+<p>
+  In <code>FineSelectionFunction</code>, stocks with the lowest market capitalization (25% of the universe) are excluded due to liquidity reasons.
```

Comment:
What reasons? Be explicit where possible
".... are excluded due to low liquidity."